lib/files.go

```diff
@@ -49,9 +45,20 @@ func CheckFileExist(file string) bool {
 
 // Unzip will decompress a zip archive, moving all files and folders
 // within the zip file (parameter 1) to an output directory (parameter 2).
-func Unzip(src string, dest string) ([]string, error) {
+// fileToUnzip (parameter 3) specifices the file within the zipfile to be extracted.
```

Comment:
Minor spelling error:
specifices = specifies

Reply:
Fixed in 6766145